src/jobs/docker-publish.yml

```diff
@@ -76,7 +76,14 @@ parameters:
       Include any additional steps to collect dependency information
       before `jfrog rt bp` is executed.
 
-executor: machine
+  docker-layer-caching:
+    type: boolean
+    default: false
+    description: "Should Docker Layer Caching be enabled for this executor"
+
+executor:
+  <<: machine
```

Comment:
Ah, yeah, I thought that `machine` was getting passed in as a variable somehow (not just a simple string), because I didn't know how all these files got stitched together and just assumed the full thing was getting slotted in somewhere along the way. TIL about FYAML and parametrized executors :)

Reply:
Yep, an orb is essentially a namespace, with its own `{jobs,commands,executors}`. The build process collects up directories and uses filenames for keys, i.e. `executors/foo.yml` becomes `{executors: {foo: {}}`.